src/navinator.js

```diff
@@ -105,7 +109,25 @@
 
             if (!_.$pageElement.hasClass('navinator-initialized')) {
                 _.$pageElement.addClass('navinator-initialized');
-                console.log('%cNavinator Initialized!', 'font-size: 1.1em; font-weight: bold; color: green');
+                if (_.options.debug) {
+                    console.log('%cNavinator Initialized!', 'font-size: 1.1em; font-weight: bold; color: green');
+                }
+            }
+
+            // Check for sub navigation
+            if (_.subNav) {
+                $(_.subNav).hide();
+                $(_.subNav).each(function (index, element) {
+                    var $parentLink = $(element).prev();
+                    var $parentEl = $(this).parent();
+                    $parentEl.addClass('navinator-has-subnav');
+                    $parentLink.on('click.subNavToggle', function (ev) {
+                        ev.preventDefault();
+                        var $parentEl = $(this).parent();
+                        $parentEl.toggleClass('navinator-sub-open');
+                        $(_.subNav, $parentEl).slideToggle(500);
+                    });
+                });
```

Comment:
New sub nav script.

- It hides all the sub nav elements.
- It goes through each sub nav element and finds it's parent and attaches a styling class of `navinator-has-subnav` in case the user wants to add a UI indicator for these menus.
- It gets the link that should activate the sub nav. Assumed it'll be a previous sibling of the sub nav element.
- Attaches click events to each sub nav's parent links.
- Click event toggles slide open/close and a styling class.